app/src/main/java/com/globant/example/mentorapp/home/presentation/view/adapter/ListUsersAdapter.java

```diff
@@ -47,19 +51,27 @@ public void onBindViewHolder(ViewHolder holder, int position) {
 
     @Override
     public int getItemCount() {
-        return users != null ? users.size() : context.getResources().getInteger(R.integer.zero_constant);
+        return users != null ? users.size() : Utilities.ZERO_CONSTANT;
     }
 
-    public static class ViewHolder extends RecyclerView.ViewHolder {
+    class ViewHolder extends RecyclerView.ViewHolder {
 
         private TextView userNameTextView;
         private ImageView userImage;
 
-        public ViewHolder(View itemView) {
+        ViewHolder(View itemView) {
             super(itemView);
             userNameTextView = (TextView) itemView.findViewById(R.id.textName);
             userImage = (ImageView) itemView.findViewById(R.id.imageUser);
+            itemView.setOnClickListener(new View.OnClickListener() {
+                @Override
+                public void onClick(View v) {
+                    listener.onUserSelected(userNameTextView.getText().toString());
```

Comment:
You can set the User instance or the Username to this ViewHolder so you avoid calling the View getter